packages/app-desktop/commands/convertNoteToMarkdown.ts

```diff
@@ -0,0 +1,45 @@
+import { _ } from '@joplin/lib/locale';
+import Note from '@joplin/lib/models/Note';
+import { stateUtils } from '@joplin/lib/reducer';
+import { CommandRuntime, CommandDeclaration } from '@joplin/lib/services/CommandService';
+import { MarkupToHtml } from '@joplin/renderer';
+import { runtime as convertHtmlToMarkdown } from '@joplin/lib/commands/convertHtmlToMarkdown';
+
+export const declaration: CommandDeclaration = {
+	name: 'convertNoteToMarkdown',
+	label: () => _('Convert note to Markdown'),
+};
+
+export const runtime = (): CommandRuntime => {
+	return {
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
+		execute: async (context: any, noteId: string = null) => {
```

Comment:
```suggestion
		execute: async (context: CommandContext, noteId: string = null) => {
```

**Types**: `CommandContext` can be imported from `@joplin/lib/services/CommandService`.